extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -1431,13 +1431,22 @@ export class ProjectsController {
 		return new CreateProjectFromDatabaseDialog(profile);
 	}
 
-	public async createProjectFromDatabaseCallback(model: ImportDataModel, connectionId?: string) {
+	public async createProjectFromDatabaseCallback(model: ImportDataModel, connectionInfo: string | mssqlVscode.IConnectionInfo | undefined) {
```

Comment:
can this still be `connectionId?` instead of adding the `| undefined`?

Reply:
Yes, updated it.